src/client/java/dev/creesch/model/WebsocketMessageBuilder.java

```diff
@@ -73,7 +79,7 @@ public static WebsocketJsonMessage createLiveChatMessage(
         long timestamp = Instant.now(Clock.systemUTC()).toEpochMilli();
         WebsocketJsonMessage.ChatServerInfo serverInfo =
             MinecraftServerIdentifier.getCurrentServerInfo();
-        String minecraftVersion = SharedConstants.getGameVersion().getName();
+        String minecraftVersion = SharedConstants.getGameVersion().name();
```

Comment:
I noticed here is using GameVersion `name()` instead of `id()`. Is it intentional?